### PR TITLE
(TK-178) Upgrade Jetty dependency to 9.2.10

### PR DESCRIPTION
**Pull request by camlow325**
This commit upgrades the tk-jetty9's Jetty dependency to 9.2.10.

### CODE REVIEW

**Comment by KevinCorcoran** [comment]
Would be good to document why ... maybe the CHANGELOG is the correct place for that.

**Comment by camlow325** [comment]
Yeah, I figured we'd document that in the CHANGELOG.md when we do the next release.

File: project.clj [hunk 1]
```diff
@@ -25,13 +25,13 @@
...
-                 [org.eclipse.jetty/jetty-servlet "9.2.8.v20150217"]
-                 [org.eclipse.jetty/jetty-servlets "9.2.8.v20150217"]
-                 [org.eclipse.jetty/jetty-webapp "9.2.8.v20150217"]
-                 [org.eclipse.jetty/jetty-proxy "9.2.8.v20150217"]
-                 [org.eclipse.jetty/jetty-jmx "9.2.8.v20150217"]
+                 [org.eclipse.jetty/jetty-servlet "9.2.10.v20150310"]
+                 [org.eclipse.jetty/jetty-servlets "9.2.10.v20150310"]
+                 [org.eclipse.jetty/jetty-webapp "9.2.10.v20150310"]
+                 [org.eclipse.jetty/jetty-proxy "9.2.10.v20150310"]
+                 [org.eclipse.jetty/jetty-jmx "9.2.10.v20150310"]
```
Comment (nwolfe) [inline project.clj]: Would it be worthwhile to `def` this version string? They're all the same, and all have to change at the same time right?

Comment (camlow325) [inline project.clj]: That thought has crossed my mind a few times.  Not sure if we did it this way just so we were more intentional about understanding what deps we were going to be pulling in.  Each time I've updated Jetty, I've downloaded the Jetty source and run mvn deps on it to confirm that all of these match up for a given release.  Maybe I've been too paranoid there.

I'm open to this change, though.  @cprice404 - what do you think?

**Comment by nwolfe** [comment]
:+1: other than minor version string comment

File: project.clj (hunk 1 above)
Comment (cprice404) [inline project.clj]: i'm fine either way

**Comment by camlow325** [comment]
I went ahead and did the common def in af64648.

**Pull request merged**